include/fuzzuf/cli/fuzzer/nautilus/build_nautilus_fuzzer_from_args.hpp

```diff
@@ -0,0 +1,217 @@
+/*
+ * fuzzuf
+ * Copyright (C) 2022 Ricerca Security
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+/**
+ * @file build_nautilus_fuzzer_from_args.hpp
+ * @brief Register CLI option for Nautilus
+ * @author Ricerca Security <fuzzuf-dev@ricsec.co.jp>
+ */
+#ifndef FUZZUF_INCLUDE_CLI_FUZZER_NAUTILUS_BUILD_NAUTILUS_FUZZER_FROM_ARGS_HPP
+#define FUZZUF_INCLUDE_CLI_FUZZER_NAUTILUS_BUILD_NAUTILUS_FUZZER_FROM_ARGS_HPP
+
+#include <iostream>
+#include "fuzzuf/algorithms/afl/afl_option.hpp"
+#include "fuzzuf/algorithms/nautilus/fuzzer/option.hpp"
+#include "fuzzuf/algorithms/nautilus/fuzzer/setting.hpp"
+#include "fuzzuf/cli/put_args.hpp"
+#include "fuzzuf/exceptions.hpp"
+#include "fuzzuf/executor/native_linux_executor.hpp"
+#include "fuzzuf/utils/filesystem.hpp"
+#include "fuzzuf/utils/optparser.hpp"
+
+
+namespace fuzzuf::cli::fuzzer::nautilus {
+
+// Fuzzer specific help
+// TODO: Provide better help message
+static void usage(po::options_description &desc) {
+    std::cout << "Help:" << std::endl;
+    std::cout << desc << std::endl;
+    exit(1);
+}
+
+}
+
+using fuzzuf::algorithm::nautilus::fuzzer::NautilusFuzzer;
+
+/**
+ * @fn
+ * @brief Build Nautilus fuzzer instance from CLI arguments
+ * @param (fuzzer_args) Arguments passed to Nautilus
+ * @param (global_options) Global options
+ */
+template <class TFuzzer, class TNautilusFuzzer>
+std::unique_ptr<TFuzzer> BuildNautilusFuzzerFromArgs(
+  FuzzerArgs &fuzzer_args,
+  GlobalFuzzerOptions &global_options
+) {
+  po::positional_options_description pargs_desc;
+  pargs_desc.add("fuzzer", 1);
+  pargs_desc.add("pargs", -1);
+
+  /* Options */
+  bool no_forksrv;
```

Comment:
```suggestion
  bool forksrv;
```
AFL and AFLFast use `forksrv` option (default to `true`, thus a boolean value is the opposite) so it'd be better to match them for the sake of clarity.

Reply:
Fixed in [17e958e](https://github.com/fuzzuf/fuzzuf/pull/23/commits/17e958eba3c05dfcef24744392a3f061ce8c2601)